sources/packages/web/src/components/common/OverawardDetails.vue

```diff
@@ -77,67 +67,60 @@
       >
     </template>
     <content-group>
-      <toggle-content :toggled="!overawardDeductions.length">
-        <DataTable
-          :value="overawardDeductions"
-          class="p-m-4"
-          :paginator="true"
-          :rows="pageLimit"
-          :rowsPerPageOptions="PAGINATION_LIST"
+      <toggle-content
+        :toggled="!overawardDeductions.length"
+        message="No overaward deductions found."
+      >
+        <v-data-table
+          :headers="overawardDeductionsHeaders"
+          :items="overawardDeductions"
+          :items-per-page="DEFAULT_PAGE_LIMIT"
+          :items-per-page-options="ITEMS_PER_PAGE"
+          :mobile="isMobile"
         >
-          <Column field="dateAdded" header="Date added">
-            <template #body="slotProps">
-              <span>
-                {{
-                  slotProps.data.dateAdded
-                    ? dateOnlyLongString(slotProps.data.dateAdded)
-                    : dateOnlyLongString(slotProps.data.createdAt)
-                }}
-              </span>
-            </template>
-          </Column>
-          <Column field="applicationNumber" header="Application #">
-            <template #body="slotProps">
-              <span>
-                {{ emptyStringFiller(slotProps.data.applicationNumber) }}
-              </span>
-            </template></Column
-          >
-          <Column field="overawardOrigin" header="Origin"></Column>
-          <Column v-if="showAddedBy" field="addedByUser" header="Added By"
-            ><template #body="slotProps">
-              <span>
-                {{ emptyStringFiller(slotProps.data.addedByUser) }}
-              </span>
-            </template></Column
-          >
-          <Column field="awardValueCode" header="Award"></Column>
-          <Column field="overawardValue" header="Amount deducted">
-            <template #body="slotProps">
-              <span>
-                {{ formatCurrency(slotProps.data.overawardValue) }}
-              </span>
-            </template></Column
-          >
-        </DataTable>
+          <template #loading>
+            <v-skeleton-loader type="table-row@5"></v-skeleton-loader>
+          </template>
+          <template #[`item.dateAdded`]="{ item }">
+            {{ formatDateAdded(item) }}
+          </template>
+          <template #[`item.applicationNumber`]="{ item }">
+            {{ emptyStringFiller(item.applicationNumber) }}
+          </template>
+          <template #[`item.overawardOrigin`]="{ item }">
+            {{ item.overawardOrigin }}
+          </template>
+          <template v-if="showAddedBy" #[`item.addedByUser`]="{ item }">
```

Comment:
The v-if is technically required with conditional headers but I left in for clarity.